### DISCUSSION TITLE
Add Calendars section

### DISCUSSION THREAD

**Discussion by einekratzekatze** [pluja/awesome-privacy#247] [Ideas]
Its super hard to find a great calender app for android. I hope you got some recommendations

**Comment by darhma** [comment]
Two good android calendar apps are simple calendar from simple mobile tools and etar

**Comment by pluja** [comment]
Will add some Calendar apps soon

**Comment by jonidimo** [comment]
I use this pdf calendar : https://github.com/kudrykv/latex-yearly-planner
